### PR TITLE
created a sound button which can play a sound effect when clicked

### PR DESCRIPTION
**Pull request by tomy204**
to be used instead of regular buttons to play a click sound effect

### CODE REVIEW

File: Scenes/Kaleidoscope/KaleidoscopeViewport.tscn [hunk 1]
```diff
@@ -5,3 +5,5 @@ anchor_right = 1.0
 anchor_bottom = 1.0
 
 [node name="Viewport" type="Viewport" parent="."]
+handle_input_locally = false
+render_target_update_mode = 3
```
Comment (Maaack) [inline Scenes/Kaleidoscope/KaleidoscopeViewport.tscn]: I'm not sure how these changes relate to the sound button? I'd prefer to remove them from the merge request if they are unrelated.

**Review by Maaack — CHANGES_REQUESTED** [review]
Rest of this looks good.

File: Scenes/Kaleidoscope/KaleidoscopeViewport.tscn (hunk 1 above)
Comment (tomy204) [reply Scenes/Kaleidoscope/KaleidoscopeViewport.tscn]: i didn't change anything in this, but i have opened the file to see what it looks like, maybe my godot added this ?
i am using 3.4.4

Comment (Maaack) [reply Scenes/Kaleidoscope/KaleidoscopeViewport.tscn]: Yeah, probably it just added it. I'll switch to 3.4.4 (from 3.5-rc6) and reload everything before I cause anyone further unexpected issues.

Comment (tomy204) [reply Scenes/Kaleidoscope/KaleidoscopeViewport.tscn]: ok, i can see there is a request change, should i add a new commit with these 2 lines removed ?

**Review by Maaack — APPROVED** [review]

**Pull request merged**